charts/app-of-apps/values-integration.yaml

```diff
@@ -112,6 +112,8 @@ datagovukHelmValues:
               "www.integration.data.gov.uk",
               "integration.data.gov.uk"
           ]}}]
+    config:
```

Comment:
Can we add it under the `config` key defined on line 93?

Reply:
sure, will move it